pkg/network/protocols/http/shared_libraries_test.go

```diff
@@ -64,11 +71,66 @@ func TestSharedLibraryDetection(t *testing.T) {
 	assert.Equal(t, fpath, pathDetected)
 }
 
+func TestSharedLibraryDetectionWithRoot(t *testing.T) {
+	t.Skip("skip for the moment as some distro are not friendly with busybox package")
+
+	tdir := t.TempDir()
```

Comment:
Use indicative names so other people can easily read the code
```suggestion
	tempDir := t.TempDir()
```